src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1675,4 +1675,55 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Increment the number of called alternate and reference plus alternate alleles for a genotype
+     *
+     * @param calledAltAlleles number of called alternate alleles for all genotypes
+     * @param calledAlleles    number of called alleles for all genotypes
+     * @param genotype         genotype
+     * @return incremented called alleles
+     * @throws IllegalArgumentException if calledAltAlleles or genotype are null
+     */
+    public static int incrementChromosomeCountsInfo(final Map<Allele, Integer> calledAltAlleles, final int calledAlleles, final Genotype genotype) {
```

Comment:
this api is sub optimal - it does not need the `calledAlleles` argument. Remove the argument and make the method just return the number (clients will then add that number to whatever they need). Rename method to `getChromosomeCounts` (there's no `Info`)